src/app/containers/NewStoryPromoList/index.jsx

```diff
@@ -0,0 +1,22 @@
+import { GEL_SPACING_DBL } from '#app/legacy/gel-foundations/src/spacings';
```

Comment:
This seems to only be use in RelatedContent is that right? If so, should it be a seperate component?

Reply:
Yeah ignore this comment I can see the multiple usages now, reusing list items may be useful mostly as they encapsulate correct a11y markup.